service/training.rest.xml

```diff
@@ -0,0 +1,27 @@
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/rest-api-2.1.xsd" name="Product" displayName="Product REST API" version="2.0.0">
+    <resource name="Product" require-authentication="anonymous-all">
+
+        <method type="get">
+            <service name="mantle.training.TrainingServices.get#Product"/>
```

Comment:
Please rethink for better naming of services wherever possible, I think this should be get#Products as it returns all products.